### ISSUE TITLE
[busy-indicator] allow setting aria-label on internal progress indicator

### ISSUE THREAD

**Issue by samrichardsontylertech** [tyler-technologies-oss/forge#255]
**Describe the bug:**
An `aria-label` is required on elements with the `progressbar` role, but the busy indicator component does not currently allow this to be set on its internal progress indicator.

**To Reproduce:**
1. Open a busy indicator with a progress indicator
2. Run an audit tool or inspect the progress indicator to see that it does not have a label

**Expected behavior:**
It should be possible to add the required `aria-label` to any instance of a progress indicator. Either the `ariaLabel` property should be included on the busy indicator as it is for the progress indicator components, or the `titleText` property should also set the `aria-label`.

**Please complete the following information:**
 - Forge version: 2.13.2
 - I have searched existing issues before creating this report? Y
 - Browser: All
 - Platform: All
 - OS: All
